charts/nominatim/values.yaml

```diff
@@ -67,6 +67,20 @@ nominatimInitialize:
 nominatimReplications:
   enabled: false
   replicationUrl: https://download.geofabrik.de/europe/germany/sachsen-updates/
+  threads: 1
```

Comment:
Is default: https://github.com/osm-search/Nominatim/blob/12dbfb077753361b9875cdb295d1be3279d97571/nominatim/tools/database_import.py#L98